distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestCatalog.java

```diff
@@ -1907,6 +1918,7 @@ public void testContentDirectoryMonitor() throws Exception {
         cdmProperties.putAll(getServiceManager().getMetatypeDefaults("content-core-directorymonitor",
                 "org.codice.ddf.catalog.content.monitor.ContentDirectoryMonitor"));
         cdmProperties.put("monitoredDirectoryPath", tmpDir.toString() + "/");
+        cdmProperties.put("processingMechanism", "delete");
```

Comment:
Interestingly enough, I found this one by accident. When the default processing mechanism was changed we didn't update the itests, so as a result we had no itest coverage for this mechanism.